src/app/_components/InputFields/TextInputField.js

```diff
@@ -14,11 +14,15 @@ export function TextInputField({
   error = '',
   required = false,
   variant = variants.default,
+  absolute = true,
 }) {
   const id = `textinput_${name}`;
+  const absoluteError = absolute ? 'absolute top-[45px] duration-150 ease-out transform' : '';
+  const absoluteLabel = absolute ? 'absolute bottom-[47px]' : '';
+
   return (
-    <div className={cn(variant.mainContainer.base)}>
-      {error && <p className={cn(variant.errorParagraph.base)}>{error}</p>}
+    <div className={cn(`${variant.mainContainer.base} relative`)}>
```

Comment:
🟠 if you're using cn utility, consider to refactor styles like this: className={cn("variant.mainContainer.base relative")}

Reply:
guys, why do you need "cn" here at all?

Reply:
@Yivgha, I apologize. But the variables:
variant.mainContainer.base
in your use case will become just a string.